packages/react/src/components/organisms/FooterSlim/index.js

```diff
@@ -16,60 +16,77 @@ import IconPhone from 'MayflowerReactBase/Icon/IconPhone';
+    'ma__footer-slim__container__inner--stacked': stackedLogo
+  });
+
+  return(
+    <footer className="ma__footer-slim" id="footer">
+      <div className="ma__footer-slim__container ma__container">
+        <div className={logoWrapperClasses}>
+          <SiteLogo />
         </div>
-        <div className="ma__footer-slim__details">
-          {props.links && (
-            <div className="ma__footer-slim__links">
-              {props.links.map((link, linkIndex) => (
-                /* eslint-disable-next-line react/no-array-index-key */
-                <a href={link.href} key={`footslimlinks-${linkIndex}`}>{link.title}</a>
-              ))}
-            </div>
-          )}
-          {props.contact && (
-            <div className="ma__footer-slim__contact">
-              {props.contact.address && (
-              <p>
-                <IconMarker width={20} height={20} />
-                <span>{props.contact.address}</span>
-              </p>
-              )}
-              {props.contact.phone
-              && (
-              <p>
-                <IconPhone width={20} height={20} />
-                <span>{props.contact.phone}</span>
-              </p>
-              )}
-              {props.contact.online && props.contact.online.href && props.contact.online.title
-              && (
-              <p>
-                <IconLaptop width={20} height={20} />
-                <a href={props.contact.online.href}>{props.contact.online.title}</a>
-              </p>
-              )}
-            </div>
-          )}
+        <div className={innerWrapperClasses}>
+          <div className="ma__footer-slim__info">
+            <div className="ma__footer-slim__title">{title}</div>
+            <p>{description}</p>
+            <p className="ma__footer-slim__copyright">
+              &copy;
+              {' '}
+              {year}
+              {' '}
+              Commonwealth of Massachusetts
+            </p>
+          </div>
+          <div className="ma__footer-slim__details">
+            {links && (
+              <div className="ma__footer-slim__links">
+                {links.map((link, linkIndex) => (
+                  /* eslint-disable-next-line react/no-array-index-key */
+                  <a href={link.href} key={`footslimlinks-${linkIndex}`}>{link.title}</a>
+                ))}
+              </div>
+            )}
+            {contact && (
+              <address className="ma__footer-slim__contact">
+                {contact.address && (
+                  <div className="ma__footer-slim__contact__item">
+                    <IconMarker width={20} height={20} />
+                    <span>{contact.address}</span>
+                  </div>
+                )}
+                {contact.phone && (
+                  <div className="ma__footer-slim__contact__item">
+                    <IconPhone width={20} height={20} />
+                    <span>{contact.phone}</span>
```

Comment:
This could potentially be an anchor tag:

```jsx
<a href={`tel:${contact.phone}`}>{contact.phone}</a>
```

Reply:
good call, will update